src/pages/CartPage.js

```diff
@@ -277,12 +288,12 @@ const CartPage = () => {
 
       dispatch(updateShopUser(data.updatedUser));
 
-      setSuccessMessage('Pedido criado com sucesso. Será contactado o mais brevemente possivel para confirmar o seu pedido. Consulte os detalhes do seu pedido no seu perfil.');
+      setSuccessMessage('Pedido criado com sucesso. Será contactado/a em breve para confirmar o seu pedido. Consulte os detalhes do seu pedido no seu perfil.');
 
       dispatch(clearCart());
     } catch (error) {
       setErrorMessage(error.message);
-      setBtnLoading(false);
+      setIsLoading(false);
```

Comment:
I usually set the loading to false in the `finally` so it will stop loading if it fails or if it succeeds, not sure if it makes sense in your case

Reply:
I think it makes sense.